rfcs/rdm-0068-data-migration.md

```diff
@@ -0,0 +1,369 @@
+| [CloverDX](http://Cloverdx.com)                             |     No      |             -             | Commercial                                                                                 |                                        |     |     |                                                 |
+| [Visual Flow](https://visual-flow.com)                      |     Yes     |            Yes            | Cannot find the docs                                                                       |                                        |     |     |                                                 |
+| [KETL](https://www.linuxlinks.com/ketl/)                    |     Yes     |             ?             | Cannot find the docs                                                                       |                                        |     |     |                                                 |
+| [Singer](https://www.singer.io/)                            |     Yes     |     No Elasticsearch      | Lightweight message passing, CLI based (No UI), poor monitoring/state management           |                                        |     |     |                                                 |
+| [Pentaho Kettle](https://github.com/pentaho/pentaho-kettle) |     Yes     |            Yes            | Oriented towards Hadoop/YARN ETL Processes                                                 |                                        |     |     |                                                 |
+| [Mara Pipelines](https://github.com/mara/mara-pipelines)    |     Yes     |            No             | Code-cenetered pipelines definition, no enforcing of framework/concept, poor documentation |                                        |     |     |                                                 |
+| [Airbyte](https://airbyte.com/)                             |     Yes     | Unclear | Focused on EL(T), Transformations via SQL queries (after loading)                          |
+| [Apeche NiFi](https://nifi.apache.org) | Yes | No, only Jython | No support for python scripts, script calls proof sub-performant. |
+
+### Performance
+
+Performance tests were carried out. The chose unit of measurement is 3M
+records, taking into account data transformation and insertion, without
+files migration. Note that the tests were not carried on the same hardware
+so they might be slightly skewed, however, the difference is still
+significative enough to justify the choice of implementation of
+Invenio-RDM-Migrator.
+
+| Method | Time 3M |
+|:------:|:-------:|
+| Apache NiFi | 2.5d |
+| InvenioRDM services | 7h30 | 
+| RDM-Migrator implementation | 4h30 | 
+
+## Resources/Timeline
```

Comment:
do we need this section? quite hard to estimate

Reply:
It was on the template, so I left it. Agree on removing.

Reply:
removed